index.js

```diff
@@ -149,10 +156,19 @@ async function linkModule(moduleName) {
       type = 'none';
       return { moduleName, type, target };
     }
+    // Check if there is a nested alias
+    if (moduleName.includes('/')) {
+      // If every directory is already made, mkdir will throw an error
+      try {
+        // We need to create every directory except the last
+        let parentDirectories = moduleName.substr(0, moduleName.lastIndexOf('/'));
+        await mkdir(path.join('node_modules', parentDirectories), { recursive: true });
+      } catch (err) {}
```

Comment:
Perhaps we could catch only the specific error we expect and rethrow otherwise:
```js
  } catch (err) {
    if (err.code !== 'EEXISTS') {
      throw err;
    }
  }
```

Reply:
Done.